CautionaryAlertsApi.Tests/V1/E2ETests/GetAlertsByPersonIdE2ETests.cs

```diff
@@ -57,7 +61,7 @@ public async Task GetAlertsByPersonIdReturnsAlertsIfTheyExist()
             var returnedAlerts = JsonConvert.DeserializeObject<CautionaryAlertsMMHPersonResponse>(data);
 
             returnedAlerts.PersonId.Should().Be(id);
-            returnedAlerts.Alerts.Should().HaveSameCount(alerts);
+            returnedAlerts.Alerts.Count.Should().Be(2);
```

Comment:
It's good that we check for count, but are we sure that we're returning the correct ones?

It's possible that some code edit in the future may introduce a bug that causes the API to return alerts like so:
``` cs
 [
      { ..., IsActive: false, ... },
      { ..., IsActive: true, ... }
 ]
```
In such event, this test wouldn't be able to tell the difference between it returning only `IsActive` `true`'s and returning the above, which is a mixture.

Reply:
Updated the fluent assert to find results where isActive is true and then check the count